### PR TITLE
add some more checkers (DNS)

### PR DESCRIPTION
**Pull request by DanielRuf**
This PR adds the checkers from the DNS section of https://github.com/Brunty/awesome-checker-services/issues/16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +22,25 @@ List of links to the various checkers out there on the web for sites, domains, s
...
-| https://www.whatsmydns.net/ | Global DNS propagation checker | Free |
+| https://dnsspy.io/ | Monitor, validate and verify your DNS configurations. | Free and paid plans |
+| https://dnswatch.info/ | DNS Lookups & Performance Monitoring. | Free |
+| http://dnscheck.pingdom.com/ | Enter a domain name to check if its DNS servers are configured correctly. | Free |
+| https://www.domainiq.com/tools | DNS, IP, MX tools. | Free and paid plans |
+| https://whois.domaintools.com/ | Reverse whois & IP lookup. | Free and paid plans |
+| https://www.tcpiputils.com/ | Reverse/history tools for DNS, IP, MX. | Free and paid plans |
+| http://www.viewdns.info/ | View DNS, IP, MX information. | Free and paid plans |
+| https://www.whatsmydns.net/ | Global DNS propagation checker. | Free |
+| http://bgp.he.net | Check IPs, networks, whois, ASNs, prefixes and more. | Free |
```
Comment (Brunty) [inline README.md]: As per the [Contribution Guide](https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md) can you add them alphabetically?

> Within a section, please ensure new services are added alphabetically (by name / domain) in the list.

Comment (DanielRuf) [inline README.md]: Ok, will do so. Thanks for the hint.

**Comment by Brunty** [comment]
Thanks for these PRs @DanielRuf :)

**Comment by DanielRuf** [comment]
If some entries are in the wrong order and I made some small mistakes you can push to my branches, "Allow edits from maintainers." is always enabled =)

Feedback is very welcome.

**Pull request merged**